doc/users-guide/base/typeAliases.rst

```diff
@@ -0,0 +1,67 @@
+:title: Users Guide: Type Aliases
+
+.. _ug-typeAliases:
+
+Type Aliases
+============
+
+In addition to :ref:`declaring variables <ug-vars>` using ``var`` and
+:ref:`constants <ug-constParam>` using ``const`` and ``param``, Chapel
+supports the declaration of *type aliases* using the ``type`` keyword.
+As the term suggests, type aliases create symbolic names for types.
```

Comment:
This sounds like a good clarification to make.  

FWIW, my current thinking on the topic is that while I'd ultimately like a means of creating new types, at this point, we're far enough into things that if we were to do that, we should probably use a different keyword than simply 'type'.  E.g.,

    typedef age: int;

as a means of saying "define a new type named 'age' that is a subtype of 'int'".

Unlike 'type', I imagine that 'typedef' would not be a legal argument intent.  By extension if a class/record contained a 'typedef' declaration, I wouldn't expect any compiler-generated initializers to support an override via argument (i.e., the presence of the typedef wouldn't make the class/record generic... only 'type's would do that).